static/bidder-params/adocean.json

```diff
@@ -4,10 +4,10 @@
   "description": "A schema which validates params accepted by the AdOcean adapter",
   "type": "object",
   "properties": {
-    "emiter": {
+    "emitterPrefix": {
       "type": "string",
-      "description": "AdOcean emiter",
-      "pattern": ".+"
+      "description": "AdOcean emitter prefix",
+      "pattern": "^[\\w\\-]+$"
```

Comment:
@mmuras  this might be a breaking change.  Consider scenario where incoming request is still using  `emitter` .  Consider having some transition period before removing `emiter` param.

Reply:
@mmuras for better safety its better to have some transition period and maintain backward compatibility  with `emiter` param.

Reply:
I really appreciate your concerns but in this case transition period will be of no value. Client's who use a emiter parameter (and let's imagine that they won't be informed by us) will either notice immediately (without transition period) or at the end of transition period. Is transition period required for this pull request to be merged?

Reply:
Prebid uses fail fast strategy. for example if incoming auction request mentions multiple bidders. One of them is AdOcean and has `emiter` as ext param then request will fail in validation phase and entire auction will get aborted. This seems unfair for other bidders who are part of auction.